File: GreenopsBackendupdate/Models/WasteReport.cs
using System.ComponentModel.DataAnnotations;

namespace Greenops.Models
{
    public class WasteReport
    {
        [Key]
        public int ReportId { get; set; }

        public int UserId { get; set; }

        public string District { get; set; }

        public string Place { get; set; }

        public string Description { get; set; }

        public string? WasteImage { get; set; }

        public DateTime ReportDate { get; set; }
        // For WasteReport
    }
}

File: GreenopsBackend_JavaSpring/src/main/java/com/greenops/entity/WasteReport.java
package com.greenops.entity;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Table(name = "WasteReports")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class WasteReport {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ReportId")
    private Integer reportId;

    @Column(name = "UserId", nullable = false)
    private Integer userId;

    @Column(name = "District", nullable = false)
    private String district;

    @Column(name = "Place", nullable = false)
    private String place;

    @Column(name = "Description", nullable = false)
    private String description;

    @Column(name = "WasteImage")
    private String wasteImage;

    @Column(name = "ReportDate", nullable = false)
    private LocalDateTime reportDate;
}